README.md

```diff
@@ -16,27 +14,26 @@ Require this package, with [Composer](https://getcomposer.org/), in the root dir
 composer require wordplate/translator
 ```
 
-Login to the WordPress administrator dashboard and active the [Polylang](https://wordpress.org/plugins/polylang/) plugin.
+Login to the WordPress administrator dashboard and activate the [Polylang](https://wordpress.org/plugins/polylang/) plugin. Go to the Polylang settings page and add at least one language.
 
 ## Usage
 
-Register the translations with the `register_translations` method.
+Register the translations with the `register_translations` method. They will now be available for translation on the 'Strings translations' page in the WordPress administrator dashboard.
 
 ```php
 register_translations([
     'general' => [
-        'general-language-english' => 'The english language',
-        'general-language-swedish' => 'The swedish language',
+        'English' => 'The english language',
```

Comment:
Is this the preferred way of registering translation keys? Coming from Laravel we are used to kebab-case syntax for translations keys.